### PR TITLE
Wicked: add teaming test cases

### PR DESCRIPTION
**Pull request by jlausuch**
Add 5 new teaming test cases.

- Related ticket: https://progress.opensuse.org/issues/50870
- Verification run: http://fromm.arch.suse.de/tests/6265

### CODE REVIEW

**Comment by jlausuch** [comment]
@asmorodskyi @cfconrad

File: tests/wicked/aggregate/ref/t12_teaming_random.pm [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Teaming, Random
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+
+use Mojo::Base 'wickedbase';
```
Comment (cfconrad) [inline tests/wicked/aggregate/ref/t12_teaming_random.pm]: Now we are using `Mojo::Base` for inheritance? I like it :+1: but wont to point @asmorodskyi to it.

**Comment by jlausuch** [comment]
> Looks nice to me!
> 
> Why is your VR with `WICKED_EXCLUDE=^t1[16]`?

I was doing test with t16 LAPC teaming test, but it's a bit more complex and will do it in a different PR.

File: tests/wicked/aggregate/ref/t12_teaming_random.pm (hunk 1 above)
Comment (jlausuch) [reply tests/wicked/aggregate/ref/t12_teaming_random.pm]: I wonder if it's really needed in REF tests with empty code :)

Comment (asmorodskyi) [reply tests/wicked/aggregate/ref/t12_teaming_random.pm]: @cfconrad  I just give up :)
@jlausuch I think we could drop a lot of ref files in a fact . we just need to come up with smarter logic which would call load_test function against same parameterized file

Comment (cfconrad) [reply tests/wicked/aggregate/ref/t12_teaming_random.pm]: I think for the empty code thingy... we could just add a "common_ref and load this one.

**Pull request merged**